src/Compilers/CSharp/Portable/Symbols/UnboundGenericType.cs

```diff
@@ -60,9 +60,9 @@ public static ImmutableArray<TypeWithAnnotations> CreateTypeArguments(ImmutableA
         public static readonly ErrorTypeSymbol Instance = new UnboundArgumentErrorTypeSymbol(string.Empty, new CSDiagnosticInfo(ErrorCode.ERR_UnexpectedUnboundGenericName));
 
         private readonly string _name;
-        private readonly DiagnosticInfo _errorInfo;
+        private readonly DiagnosticInfo? _errorInfo;
```

Comment:
>DiagnosticInfo? [](start = 25, length = 15)

Is this change necessary?

Reply:
The change is correct (some instances are created with `null`) but not necessary. I'll undo so this PR is more focused.